lib/taskana-core/src/test/java/acceptance/task/UpdateTaskAccTest.java

```diff
@@ -42,7 +43,35 @@
       userName = "user_1_1",
       groupNames = {"group_1"})
   @Test
-  void testUpdatePrimaryObjectReferenceOfTask()
+  public void should_updatePlanned_when_onlyDueWasChanged() throws Exception {
```

Comment:
These two tests would be better placed in ServiceLevelPriorityAccTest, don't you think?

Reply:
Done